Shokofin/Sync/UserDataSyncManager.cs

```diff
@@ -444,7 +454,14 @@ private async Task SyncVideo(Video video, UserConfiguration userConfig, SyncDire
                 return;
             }
             var localUserStats = UserDataManager.GetUserData(userConfig.UserId, video);
-            var remoteUserStats = await APIClient.GetFileUserStats(fileId, userConfig.Token);
+            UserStats remoteUserStats;
+            try {
+                remoteUserStats = await APIClient.GetFileUserStats(fileId, userConfig.Token);
+            }
+            catch (HttpRequestException ex) when (ex.StatusCode is HttpStatusCode.Unauthorized) {
+                Logger.LogError(ex, "Invalid or expired API token used. In the plugin settings, please reset the connection to Shoko Server in the user settings section (Jellyfin User={Username})", UserManager.GetUserById(userConfig.UserId).Username);
+                return;
```

Comment:
Since we're now silently returning instead of throwing then as a result we will have a log full of errors if it throws a 401 instead of logging once, then throwing, then marking the task as failed in the UI and halting the task. If the purpose of this PR 

```suggestion
                throw;
```